Library/Formula/task.rb

```diff
@@ -23,4 +23,12 @@ def install
   test do
     system "#{bin}/task", "--version"
   end
+
+  def caveats; <<-EOS.undent
+    In order to use the completions for the fish shell, extend $fish_complete_path with:
+      #{HOMEBREW_PREFIX}/share/doc/task/scripts/fish/
+    e.g. by putting this line into ~/.config/fish/config.fish:
+      set -x fish_complete_path #{HOMEBREW_PREFIX}/share/doc/task/scripts/fish/ $fish_complete_path
+    EOS
+  end
```

Comment:
I had to read several documentation pages to find out where the completion for task resides and how to add it to fish, since there is no automatic installation possibility like for bash or zsh. So this might someone. Where should such information be out otherwise?